helpers/vendor.js

```diff
@@ -34,8 +44,7 @@ module.exports.makeOrder = function(ingredientId,vendorId,numUnits){
         let newName = ing['name'];
         let nativeUnit = ing['nativeUnit'];
         let unitsPerPackage = ing['unitsPerPackage'];
-        //TODO make amount on package volume
-        let amount = numUnits+parseFloat(ing['amount']);
+        let amount = numUnits*spaceMapping[package]+parseFloat(ing['amount']);
```

Comment:
This is should be `numUnits*ing['unitsPerPackage'] + parseFloat(ing['amount'])`

I handle the inventory space stuff in the `IngredientHelper.updateIngredient` method.

Reply:
alright. fixed this